openapi/advertising/advertising.yaml

```diff
@@ -349,14 +351,126 @@ paths:
       description: 'Used solely for [CORS](https://developer.mozilla.org/en-US/docs/Web/HTTP/CORS) the OPTIONS request returns the list of possible HTTP methods and other headers that browsers use to protect user''s security. You should not call this operation directly. '
       tags:
         - Options
+  /campaignStats:
+    get:
+      summary: List Campaign Stats
+      tags:
+        - Campaign Stats
+      responses:
+        '200':
+          description: OK
+          content:
+            application/vnd.api+json:
+              schema:
+                type: object
+                properties:
+                  data:
+                    type: array
+                    items:
+                      $ref: '#/components/schemas/campaignStats'
+                  links:
+                    type: object
+                    properties:
+                      self:
+                        type: string
+      operationId: get-campaignStats
+      description: Retrieve stats for all connected advertising campaigns for a given connected advertising account.
+      security:
+        - OAuth2:
+            - advertising
+      parameters:
+        - $ref: '#/components/parameters/Authorization'
+        - $ref: '#/components/parameters/Content-Type'
+        - $ref: '#/components/parameters/startAt'
+        - $ref: '#/components/parameters/endAt'
+        - schema:
+            type: string
+          in: query
+          name: connectedAccount.id
```

Comment:
```suggestion
          name: filter[connectedAccount.id]
```

Reply:
This parameter isn't really a filter.  In order to get the campaign stats the we need the ID of the connected account, so this parameter is required for every request.  Since it isn't optionally filtering some larger set it doesn't make sense to label it as a filter parameter.

Reply:
I see where you are coming from. From the perspective of a partner calling the API, the `/connectedAccounts` collection contains all of the connected accounts in existence. They are requesting only the ones where the connectedAccount.id matches the value that they provided. It is a required filter. 

[Here](https://jsonapi.org/recommendations/#:~:text=it%20is%20helpful%20to%20consider%20that%20all%20of%20its%20resources%20exist%20in%20a%20single%20%E2%80%9Creference%20document%E2%80%9D%20in%20which%20each%20resource%20is%20addressable%20at%20a%20unique%20path.%20Resources%20are%20grouped%20by%20type%20at%20the%20top%20level%20of%20this%20document.) is the related JSON:API recommendation.